MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/InstrumentWidget.h

```diff
@@ -0,0 +1,236 @@
+#ifndef INSTRUMENTWIDGET_H_
+#define INSTRUMENTWIDGET_H_
+
+#include "WidgetDllOption.h"
+#include "MantidGLWidget.h"
+#include "BinDialog.h"
+#include "InstrumentWidgetTypes.h"
+
+#include "MantidQtAPI/GraphOptions.h"
+#include "MantidQtAPI/WorkspaceObserver.h"
+#include "MantidQtAPI/MantidAlgorithmMetatype.h"
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/AlgorithmObserver.h"
+
+#include <string>
```

Comment:
I'm pretty sure you could get away without this too. @LamarMoore could you please check all includes introduced.